src/components/home/components/styles.css

```diff
@@ -213,3 +213,9 @@ img {
 .search {
     margin-top: 70px;
 }
+
+.contactUs {
+    top: 350pt;
+    position: fixed;
+    transform: translateX(-100px)
+}
```

Comment:
I would prefer that you use ant d affix (https://ant.design/components/affix/) to get this done instead, given that transform might have different behavior on different browsers and devices